signac/core/synced_collections/synced_collection.py

```diff
@@ -128,10 +128,19 @@ class SyncedCollection(Collection):
     _validators: List[Callable] = []
     # Backends that support threading should modify this flag.
     _supports_threading: bool = False
+    _LoadSaveType = _LoadAndSave
 
     def __init__(self, parent=None, *args, **kwargs):
-        self._parent = parent
-        self._suspend_sync_ = 0
+        if parent is not None:
+            self._parent = parent._parent if parent._parent is not None else parent
```

Comment:
This might be silly, but it took me a little time to understand. This line makes it _seem_ as if you're accessing the "grandparent" whereas you're really accessing the top level ancestor (by construction, since the parent's `_parent` actually refers to the top level ancestor by this same logic). Would it make sense to change the name of this attribute to something like `_ancestor` or `_root` or `_root_synced_collection`, or at least add a comment to explain this fact? I realize that might require changes in several places, but just wanted to raise that for consideration.

Reply:
Thinking some more about this, the name `_parent` seems semantically inaccurate since it doesn't refer to "one level up." The name `_root` makes more sense to me.

Reply:
Yes, this is a recursive construction to flatten the parent hierarchy that we currently store. There's really no reason to know the parent, you only ever need to know the top level, so you might as well store the root. I agree that changing the name makes sense.